src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFilesSummaryRepository.java

```diff
@@ -13,7 +13,7 @@
     @Query(
         nativeQuery = true,
         value = "SELECT "
-            + "Cast(envelope.id as varchar) as envelopeId, "
+            + "payment.documentcontrolnumber, "
```

Comment:
why envelope.id removed ?

Reply:
Its easy to get payment with document control number rather than envelope.id without using envelopeRepository